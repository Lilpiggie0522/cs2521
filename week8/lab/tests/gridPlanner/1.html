i is 7
Number of vertices: 8
Number of edges: 28
Edge 0 - 1: 180.000000
Edge 0 - 2: 290.000000
Edge 0 - 3: 476.000000
Edge 0 - 4: 705.000000
Edge 0 - 5: 806.000000
Edge 0 - 6: 174.000000
Edge 0 - 7: 62.000000
Edge 1 - 2: 166.000000
Edge 1 - 3: 452.000000
Edge 1 - 4: 717.000000
Edge 1 - 5: 681.000000
Edge 1 - 6: 334.000000
Edge 1 - 7: 117.000000
Edge 2 - 3: 313.000000
Edge 2 - 4: 588.000000
Edge 2 - 5: 522.000000
Edge 2 - 6: 385.000000
Edge 2 - 7: 240.000000
Edge 3 - 4: 275.000000
Edge 3 - 5: 462.000000
Edge 3 - 6: 446.000000
Edge 3 - 7: 461.000000
Edge 4 - 5: 590.000000
Edge 4 - 6: 620.000000
Edge 4 - 7: 705.000000
Edge 5 - 6: 853.000000
Edge 5 - 7: 761.000000
Edge 6 - 7: 227.000000

Number of vertices: 8
Number of edges: 7
Edge 0 - 6: 174.000000
Edge 0 - 7: 62.000000
Edge 1 - 2: 166.000000
Edge 1 - 7: 117.000000
Edge 2 - 3: 313.000000
Edge 3 - 4: 275.000000
Edge 3 - 5: 462.000000

edge 0,6
edge 0,7
edge 1,2
edge 1,7
edge 2,3
edge 3,4
edge 3,5
<html>
<head>
  <title>Electrical Grid Plan</title>
  <link rel="stylesheet" href="./style.css">
</head>
<body>
  <h1>Electrical Grid Plan</h1>
  <p>Power plants are coloured yellow.</p>
  <div id="cy"></div>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/cytoscape/3.19.0/cytoscape.min.js" integrity="sha512-TOWs340KHbJjY/a2SCtsUcXYBg7/xPX72NKpJ3VITogkHJTy2yMyoJE0pxJjumMGHCg46ud89KO5q1Toe3Aeaw==" crossorigin="anonymous"></script>
  <script>
    var elems = [
      {data: {id: "Sydney"}, position: {x: 870, y: -204}},
      {data: {id: "Melbourne"}, position: {x: 727, y: -94}},
      {data: {id: "Adelaide"}, position: {x: 581, y: -174}},
      {data: {id: "Alice Springs"}, position: {x: 474, y: -469}},
      {data: {id: "Darwin"}, position: {x: 406, y: -736}},
      {data: {id: "Perth"}, position: {x: 65, y: -254}},
      {data: {id: "Brisbane"}, position: {x: 910, y: -374}},
      {data: {id: "Canberra"}, position: {x: 821, y: -165}, classes: ['power-plant']},
      {data: {id: 'e1', source: "Sydney", target: "Brisbane"}},
      {data: {id: 'e2', source: "Sydney", target: "Canberra"}},
      {data: {id: 'e3', source: "Melbourne", target: "Adelaide"}},
      {data: {id: 'e4', source: "Melbourne", target: "Canberra"}},
      {data: {id: 'e5', source: "Adelaide", target: "Alice Springs"}},
      {data: {id: 'e6', source: "Alice Springs", target: "Darwin"}},
      {data: {id: 'e7', source: "Alice Springs", target: "Perth"}},
    ];
  </script>
  <script src="./visualiser.js"></script>
</body>
</html>
