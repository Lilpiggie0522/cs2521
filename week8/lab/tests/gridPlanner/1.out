i is 7
Number of vertices: 8
Number of edges: 28
Edge 0 - 1: 180.000000
Edge 0 - 2: 290.000000
Edge 0 - 3: 476.000000
Edge 0 - 4: 705.000000
Edge 0 - 5: 806.000000
Edge 0 - 6: 174.000000
Edge 0 - 7: 62.000000
Edge 1 - 2: 166.000000
Edge 1 - 3: 452.000000
Edge 1 - 4: 717.000000
Edge 1 - 5: 681.000000
Edge 1 - 6: 334.000000
Edge 1 - 7: 117.000000
Edge 2 - 3: 313.000000
Edge 2 - 4: 588.000000
Edge 2 - 5: 522.000000
Edge 2 - 6: 385.000000
Edge 2 - 7: 240.000000
Edge 3 - 4: 275.000000
Edge 3 - 5: 462.000000
Edge 3 - 6: 446.000000
Edge 3 - 7: 461.000000
Edge 4 - 5: 590.000000
Edge 4 - 6: 620.000000
Edge 4 - 7: 705.000000
Edge 5 - 6: 853.000000
Edge 5 - 7: 761.000000
Edge 6 - 7: 227.000000

Number of vertices: 8
Number of edges: 7
Edge 0 - 6: 174.000000
Edge 0 - 7: 62.000000
Edge 1 - 2: 166.000000
Edge 1 - 7: 117.000000
Edge 2 - 3: 313.000000
Edge 3 - 4: 275.000000
Edge 3 - 5: 462.000000

edge 0,6
edge 0,7
edge 1,2
edge 1,7
edge 2,3
edge 3,4
edge 3,5
City Sydney at (870, 204)
City Melbourne at (727, 94)
City Adelaide at (581, 174)
City Alice Springs at (474, 469)
City Darwin at (406, 736)
City Perth at (65, 254)
City Brisbane at (910, 374)
Power Plant Canberra at (821, 165)
Power line between Sydney (870, 204) and Brisbane (910, 374)
Power line between Sydney (870, 204) and Canberra (821, 165)
Power line between Melbourne (727, 94) and Adelaide (581, 174)
Power line between Melbourne (727, 94) and Canberra (821, 165)
Power line between Adelaide (581, 174) and Alice Springs (474, 469)
Power line between Alice Springs (474, 469) and Darwin (406, 736)
Power line between Alice Springs (474, 469) and Perth (65, 254)
